public/app/features/dashboard-scene/serialization/transformSceneToSaveModelSchemaV2.ts

```diff
@@ -189,6 +189,12 @@ function getElements(scene: DashboardScene) {
         }).filter(([_, value]) => value !== undefined)
       );
 
+      // if mappings is empty, we don't want to persist it because backend will return
+      // not persist empty mappings and we'll have an unneeded diff in the save UI
+      if (!vizPanel.state.fieldConfig.defaults.mappings?.length) {
```

Comment:
I don't understand why this is happening. Probably we could solve it adding as a default value empty array in the cue schema 🤔

Reply:
See https://raintank-corp.slack.com/archives/C07SB1G4BC1/p1742910121580999